R/check-cols-complete.R

```diff
@@ -0,0 +1,50 @@
+#' Check for complete columns
+#'
+#' Check for complete columns in the data and warn (or fail) if present.
+#'
+#' @param data Data to check
+#' @param required_cols The required columns to check for completeness.
```

Comment:
Could we specify what format this should be in, to help the user? e.g. "A character vector of the required columns..."

Reply:
Thank you for the suggestion.